src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -11,6 +11,7 @@
 
 namespace Symfony\Component\Filesystem;
 
+use RuntimeException;
```

Comment:
Even better, as we have exception classes for this component, they should be used instead.